embedded-bins/kubernetes/Dockerfile

```diff
@@ -25,8 +25,11 @@ ARG TARGET_OS \
   BUILD_GO_LDFLAGS_EXTRA \
   KUBERNETES_BINS
 
+COPY riscv64.patch .
+
 RUN \
   set -ex; \
+  git apply < riscv64.patch; \
```

Comment:
Doh! This breaks the inttests, as the Kubernetes working directory is dirty now: https://github.com/k0sproject/k0s/actions/runs/15024623406/job/42229718069?pr=5848#step:11:427

 I'm thinking if we should use a tarball instead of a git checkout here?

Reply:
I'm testing `export KUBE_GIT_TREE_STATE=clean`